packages/styles/scss/patterns/sections/card-section/_card-section.scss

```diff
@@ -27,6 +27,18 @@
       background: $hover-ui;
     }
   }
+  .#{$prefix}--card--inverse {
```

Comment:
As we discussed earlier there is something not right with the icon...

This is what it looks like in the card section on white theme
![image](https://user-images.githubusercontent.com/3793636/75506212-e2087300-59a2-11ea-9971-ed60d4e12e67.png)

This is what the inverse card sub-pattern looks like on white theme 
![image](https://user-images.githubusercontent.com/3793636/75506201-d61cb100-59a2-11ea-8ec7-a23e832c2607.png)

We need to make sure we're using the same tokens.

> side note on theming we need to strategically look at theming across the board similar too. These overwrites like this aren't very scalable.

Reply:
done.